packages/application/src/requests.tsx

```diff
@@ -82,7 +109,18 @@ export class RequestHandler {
       displayWaitDialog
     );
   }
-
+  /**
+   * Make an request to the jupyterlab server.
+   *
+   * @param requestExt - The url for the request.
+   *
```

Comment:
the `Ext` here really made me confused, could we make a better name here? maybe requestURL, endpoint, servicePath...

Reply:
this part of the url is traditionally refereed to as the `path` so how about I update it to `requestPath`?